### PR TITLE
Fix create from transformation V2 config

### PR DESCRIPTION
**Pull request by pivnicek**
Mainly this was missing the datadir test, so wasn't tested

### CODE REVIEW

File: src/ConfigValidator.php [hunk 1]
```diff
@@ -134,21 +134,20 @@ private function validateTransformationV2Config(): void
...
-            throw new InvalidInputException('Configuration Row not found.', self::CONFIGURATION_INVALID);
+        if (empty($configData['configuration'])) {
+            throw new InvalidInputException('Configuration not found.', self::CONFIGURATION_INVALID);
         }
         $processor = new Processor();
         try {
             $configData = $processor->processConfiguration(
                 new TransformationV2Config(),
-                ['configuration' => $configData]
+                ['configuration' => $configData['configuration']]
```
Comment (JakubMatejka) [inline src/ConfigValidator.php:150]: Um, and this was a bug, right? I just now can't evaluate if there is supposed to be just the `configuration` or whole $configData. ;o)

Comment (pivnicek) [reply src/ConfigValidator.php:150]: yes, this was the bug that I posted on the slack.  it should be just the `configuration` of the config :-)

Comment (JakubMatejka) [reply src/ConfigValidator.php:150]: ok 😄

**Review by JakubMatejka — APPROVED** [review]
👍

**Pull request merged**